### PR TITLE
Update tensorflow-core-platform to 0.3.1

### PR DESCRIPTION
**Pull request by regadas**

### CODE REVIEW

**Comment by nevillelyh** [comment]
@regadas Should this target 0.11.0?

**Comment by regadas** [comment]
Hi @nevillelyh yea, we can just keep this here or start a branch for it, wdyt?

File: build.sbt [hunk 1]
```diff
@@ -1155,7 +1156,8 @@ lazy val site: Project = project
     `scio-sql`
   )
 
-// =======================================================================
+// 
+==================================================================
```
Comment (nevillelyh) [inline build.sbt:1160]:
```suggestion
```

File: build.sbt [hunk 2]
```diff
@@ -1155,7 +1156,7 @@ lazy val site: Project = project
     `scio-sql`
   )
 
-// =======================================================================
+//
```
Comment (nevillelyh) [inline build.sbt:1159]:
```suggestion
// =======================================================================
```

File: build.sbt [hunk 3]
```diff
@@ -89,7 +89,7 @@ val metricsVersion = "3.2.6"
 val nettyVersion = "4.1.52.Final"
 val nettyTcNativeVersion = "2.0.34.Final"
 val opencensusVersion = "0.28.0"
-val parquetExtraVersion = "0.4.0"
+val parquetExtraVersion = "0.4.1"
```
Comment (nevillelyh) [inline build.sbt:92]:
```suggestion
val parquetExtraVersion = "0.4.2"
```

**Comment by nevillelyh** [comment]
Reworked in #3971

**Pull request closed without merging**